coral/src/app/pages/topics/schema-request.test.tsx

```diff
@@ -25,6 +30,7 @@ describe("SchemaRequest", () => {
     beforeAll(() => {
       mockGetSchemaRegistryEnvironments.mockResolvedValue([]);
       mockCreateSchemaRequest.mockImplementation(jest.fn());
+      mockGetTopicNames.mockResolvedValue([]);
```

Comment:
Curious about this addition, how come the test didn't fail if it was missing this mock? :o

Did one of the lib update make it fail (correctly)?

Reply:
No, the fail was unrelated to the libs, it also did only happen on CI, locally everything worked without problems. 
The test didn't fail because it's not relevant for this test if the api call was successful -> the missing API mock will  return an error,  but Jest is usually already done with the tests by then, so it won't show up or exit the test run. I think it bubbled up here because the CI ran slower than usual, which gave the error actual time to show up. 

It's a familiar problem for me, I wrestled with that in another code base already(*). Identifying actual API calls being made in these component tests  is way harder then it should be 😅 I found a way in the past to prevent them, but our use of msw mocks prevents from implementing that right now. 


(*) you could say it's my second test nemesis beside console output in tests 🙎‍♀️ 🙎‍♀️ 🙎‍♀️